src/store/actions.ts

```diff
@@ -0,0 +1,82 @@
+// @ts-ignore
+import api from '@molgenis/molgenis-api-client'
+
+export default {
+  loadTreeStructure ({ commit } : any) {
```

Comment:
maybe have a look a async wait

Reply:
This part of the code will be refactored later in simple data requests and a smart getter function